cpp/attrib.cpp

```diff
@@ -247,4 +247,18 @@ NAN_METHOD(vertexAttribPointer) {
 }
 
 
+NAN_METHOD(vertexAttribIPointer) {
+	
+	REQ_UINT32_ARG(0, indx);
+	REQ_INT32_ARG(1, size);
+	REQ_INT32_ARG(2, type);
+	REQ_UINT32_ARG(3, stride);
+	REQ_INT32_ARG(4, ptr);
```

Comment:
I copied this behavior from the `vertexAttribPointer` implementation just above this. `offset` is probably a better name instead of `ptr`, but I try not to impose my naming preferences on existing code bases :-)

Reply:
offset kind of hints towards `REQ_OFFS_ARG`